BUILD.bazel

```diff
@@ -12,3 +13,10 @@ js_library(
         "//:node_modules/@sourcegraph/prettierrc",
     ],
 )
+
+# TODO: replace npm_link_package() targets with pnpm workspaces
+npm_link_package(
+    name = "node_modules/@sourcegraph/extension-api-types",
+    src = "//client/extension-api-types",
+    package = "@sourcegraph/extension-api-types",
+)
```

Comment:
What's the purpose of this?

Reply:
Based on what Greg said in the call I'm going to drop this...